taskgraph/Task.py

```diff
@@ -1407,11 +1319,10 @@ def _get_file_stats(
                     not os.path.isdir(norm_path) or
                     not ignore_directories) and os.path.exists(norm_path):
                 if hash_algorithm == 'exists':
-                    yield (norm_path, 'exists', 'exists')
+                    yield (norm_path, 'exists')

```

Comment:
Could you update the docstring to reflect the updated tuple structure?

Reply:
Updated in 1d5314f